_layouts/product.html

```diff
@@ -181,7 +181,7 @@ <h1>{{ page.title }}</h1>
 
 <p>
   You can submit an improvement to this page
-  <a href="https://github.com/endoflife-date/endoflife.date/blob/master/{{page.path}}" title="Click the Pencil, the link takes you directly to the correct page">
+  <a href="/{{page.permalink}}/_edit" title="Click the Pencil, the link takes you directly to the correct page">
```

Comment:
The edit link on the bottom of https://deploy-preview-3084--endoflife-date.netlify.app/almalinux is: `https://almalinux/_edit`

Perhaps we need to remove the trailing slash:
```suggestion
  <a href="{{page.permalink}}/_edit" title="Click the Pencil, the link takes you directly to the correct page">
```